remi/server.py

```diff
@@ -631,18 +651,21 @@ def _get_static_file(self, filename):
         key = filename[:__i]
         path = filename[__i+1:]
         key = key.replace("/","")
-        paths = {'res': os.path.join(os.path.dirname(__file__), "res")}
-        static_paths = self._app_args.get('static_file_path', {})
-        if not type(static_paths)==dict:
-            self._log.error("App's parameter static_file_path must be a Dictionary.", exc_info=False)
-            static_paths = {}
-        paths.update(static_paths)
+        paths = self.all_paths()
         if not key in paths:
             return None
         return os.path.join(paths[key], path)
+    
+    def _overload(self, data, **kwargs):
+        """Used to overload the content before sent back to client"""
+        return data
 
-    def _process_all(self, func):
+    def _process_all(self, func, **kwargs):
         self._log.debug('get: %s' % func)
+        if 'overload' not in kwargs:
```

Comment:
Why is it required to pass the overload function as arguments? The App class is inherited, so the function **_overload** is already overloaded. Doesn't it?

Reply:
You are perfectly right, this is not needed, we just need to use self._overload to call the function (it is a scorie which need to be cleaned-up: I make  the correction and send new a new version.